setup.cfg

```diff
@@ -7,5 +7,9 @@ omit =
      node_modules/*
      .venv/*
 
+[coverage:report]
+exclude_lines = 
+     def configure_logging
+
```

Comment:
I don't see a great way to truly test configure_logging so seems reasonable to exclude it from coverage calculation